### PR TITLE
Clarified that for enhancements the affects version should normally be empty

### PR DESCRIPTION
**Pull request by calnan**
Unlike for bugs, for enhancements the affectsversion is less relevant as vast majority will be fixed in a future release and will not be backported. There are situations where it is relevant, if the code is only in an LTS release for example.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace

### Reviewers
 * [Jesper Wilhelmsson](https://openjdk.org/census#jwilhelm) (@JesperIRL - **Reviewer**)
 * [Iris Clark](https://openjdk.org/census#iris) (@irisclark - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/guide.git pull/115/head:pull/115` \
`$ git checkout pull/115`

Update a local copy of the PR: \
`$ git checkout pull/115` \
`$ git pull https://git.openjdk.org/guide.git pull/115/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 115`

View PR using the GUI difftool: \
`$ git pr show -t 115`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/guide/pull/115.diff">https://git.openjdk.org/guide/pull/115.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/guide/pull/115#issuecomment-1702114895)

### CODE REVIEW

File: src/guide/jbs-jdk-bug-system.md [hunk 1]
```diff
@@ -163,9 +164,10 @@ Now that the issue is in the right component and has the basic information, the
    * An approach that has been used for getting a consistent view of priority is to consider three aspects of the issue: **Impact** of the issue; **Likelihood** of it occurring; and, whether there is a **Workaround**. The higher the impact and likelihood the higher the priority; then, having a workaround reduces that priority - but mostly where the impact and likelihood aren't that severe.
 1. Ensure the [Affects Version/s]{.jbs-field} field is correct (within reason).
    * This may involve reproducing the bug, if doing so is fast and easy.
-   * In addition to the version where the bug was found, take special care to also investigate if the bug affects the generally supported releases - e.g. the latest "LTS" release and the latest six-month release.
+   * In addition to the version where the bug was found, take special care to also investigate if the bug affects the mainline.
```
Comment (JesperIRL) [inline src/guide/jbs-jdk-bug-system.md:167]: I see "mainline" as a name rater than a thing, so I wouldn't have "the" in front of it in this sentence.

**Review by JesperIRL — CHANGES_REQUESTED** [review]

File: src/guide/jbs-jdk-bug-system.md (hunk 1 above)
Comment (calnan) [reply src/guide/jbs-jdk-bug-system.md]: yes good point, makes sense - have removed the offending "the"

**Review by JesperIRL — APPROVED** [review]

**Pull request closed without merging**